dist/nginx.conf

```diff
@@ -154,6 +128,9 @@ server {
     proxy_cache_use_stale error timeout updating http_500 http_502 http_503 http_504;
     add_header X-Cached $upstream_cache_status;
 
+    # data: for images is needed to have the "boost" button
+    add_header Content-Security-Policy "default-src 'self'; img-src 'self' data:; script-src 'self'; object-src 'none'; style-src 'self'; form-action 'self'";
```

Comment:
Mastodon handles the CSP for these paths itself, they should not be set in nginx.

Reply:
Sorry for the delay!

The CSP issues were actually because of a recently-introduced bug, it should be fixed by https://github.com/mastodon/mastodon/pull/26105

Reply:
No worries, I was unable to work on this anyway.

Shall I make a new branch for the ciphers and then close this PR?

Reply:
Yes, please, although I am not sure how much of a good idea it is to hardcode those.

Reply:
Done: https://github.com/mastodon/mastodon/pull/26349